Automattic/jetpack
# [not verified] Revert "Restore phpcs-changed checks (#13931)"

#### Changes proposed in this Pull Request:
This reverts commit e8e83399444da5a9def0c4b56811b48cd4b8323b.

e8e83399444da5a9def0c4b56811b48cd4b8323b resulted in symlinks in the production build, which breaks the zips provided by the beta builder.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
No: bug fix.

#### Testing instructions:
1. `yarn clean-composer`
2. `yarn build-production-php`

Prior to this patch, see symlinks in vendor/automattic.

After this patch, see real packages in vendor/automattic.

#### Proposed changelog entry for your changes:
None required.
